src/test/java/core/basesyntax/write/WriteToFileTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.write;
+
+import org.junit.Assert;
+import org.junit.Test;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class WriteToFileTest {
+
+    @Test
+    public void standartWriteTest() {
```

Comment:
```suggestion
    public void standardWriteTest() {
```
назва тесту не зовсім інформативна
Спробуй деталізувати її

Reply:
shouldCorrectWritingTest